CONTRIBUTING.md

```diff
@@ -242,6 +242,10 @@ who you are:
 $ git config --global user.name "J. Random User"
 $ git config --global user.email "j.random.user@example.com"
 ```
+Be sure this local email is also added to your
```

Comment:
`s/Be/Please make` — I think that's a bit nicer but this is a super minor nit.

Reply:
Done.